test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -635,22 +635,36 @@ def addSuccess(self, test):
     def _cleanup(self):
         self.logger.info('Cleaning up temporary files')
         du = DshUtils()
-        tmpdir = tempfile.gettempdir()
-        ftd = []
-        if tmpdir:
+        dirlist = set(['/tmp', '/var/tmp'])
+        # get tmp dir from the environment
+        for envname in 'TMPDIR', 'TEMP', 'TMP':
+            dirname = os.getenv(envname)
+            if dirname: 
+                dirlist.add(dirname)
+
+        for tmpdir in dirlist:
```

Comment:
Add comment of what you are trying to do here.

Reply:
done